aws-lc-rs/src/lib.rs

```diff
@@ -292,12 +320,17 @@ mod tests {
     #[test]
     fn test_fips() {
         assert!({ crate::try_fips_mode().is_err() });
+        assert!({ crate::try_fips_cpu_jitter_entropy().is_err() });
     }
 
     #[test]
     // FIPS mode is disabled for an ASAN build
```

Comment:
Just curious is there some issue the Rust compiler/AWS-LC FIPS/ASAN? ASAN works with the FIPS build of the C library.

Reply:
With ASAN enabled [`FIPS_mode` returns 0](https://github.com/aws/aws-lc/blob/94daeb22e78465e53f97417b4c7397dbf2f6b969/crypto/fipsmodule/self_check/fips.c#L24) instead of 1.